docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst

```diff
@@ -199,6 +199,43 @@ At the prediction stage, a multi-class classifier has the following parameters:
      - ``defaultDense``
      - 1.0e-12
      - The prediction accuracy.
+   * - ``resultsToEvaluate``
+     - ``voteBased``
+     - ``computeClassLabels``
+     - The 64-bit integer flag that specifies which extra characteristics of the decision function to compute.
+       
+       Provide one of the following values to request a single characteristic 
+       or use bitwise OR to request a combination of the characteristics:
+
+       - ``computeClassLabels`` for `prediction`
+       - ``computeDecisionFunction`` for `decisionFunction`
+
+Output
+------
+
+In addition to classifier output, multiclass classifier calculates the result described below.
+Pass the ``Result ID`` as a parameter to the methods that access the result of your algorithm.
+For more details, see :ref:`algorithms`.
+
+.. list-table::
+   :widths: 10 60
+   :header-rows: 1
+   :align: left
+
+   * - Result ID
+     - Result
+   * - ``decisionFunction``
+     - A numeric table of size :math:`n \times \frac{K(K-1)}{2}` containing the results of the decision function
```

Comment:
[Here ](http://oneapi-src.github.io/oneDAL/dev_guide/algorithms/svm_multi_class/multi-class-classifier.html#training-stage) - stage use slightly different style 
![image](https://user-images.githubusercontent.com/17429885/94662034-82938c80-0310-11eb-9405-e6dec1aea301.png)

I would suggest aligning to either the first case or the second

Reply:
done